rebar.config

```diff
@@ -9,5 +9,5 @@
 {reset_after_eunit, true}.
 
 {deps, [
-        {lager, "2.0.3", {git, "git://github.com/basho/lager", {tag, "2.0.3"}}}
+        {lager, "2.*", {git, "git://github.com/basho/lager", {tag, "2.1.1"}}}
```

Comment:
Confirmed that `2.1.1` is allowed and `1.2.0` is excluded by diff of 422bc5126920dbcdf7ebecfdda34f91eac72ffb3 .